script/Ciel/Day 3/03_01_CIEL03_3_3B.txt

```diff
@@ -76,32 +82,33 @@ Senpai's hand reaches out.
 [sha:68e7dbfa48953f2e7130a9a7bc9e1d407dad7d37]{
 -- Page 197, Offset 5353.
 -- すれ違った俺の肩を呼び止めるように。
-As if trying to stop me as I pass by.
+She goes to grab my shoulder as I pass her by.
 }
 [sha:cb613451f2a7140472b12bf407f02c96c19c5030]{
 -- Page 197, Offset 5354.
 -- それを振りほどこうとして、
-I try to shake it off―――
+When I try to shake her off―――
 }
 [sha:67d9cae6e82c27cd9eb8e0a3ed4a7ced04800a87]{
 -- Page 198, Offset 5355.
 -- 何の悪意もない偶然で、世界の在り方が一変した。
-Through mere coincidence, without any malice, the way of the world changes completely.
+Through no fault of anyone's, the world is turned on its head.
 }
 [sha:fc8c939775928b159372a3d43b15349ad9ca7e95]{
 -- Page 199, Offset 5356.
 -- 『――――――、あ』
-"――――――Ah."
+'――――――%{i}Ah%{/i}.'
 }
 [sha:4ae4840fce35af51972ed7a21c7ffa9fc6937bf5]{
 -- Page 199, Offset 5357.
 -- ズレる。ズレる。ズレる。ズレる。
-The world is contorted. Contorted. Contorted. Contorted.
+Disarray. Disorder. Dislocation. Disruption.
```

Comment:
I mean, I'm not gonna fight this liberty even though I personally wouldn't take it, but "disarray" and "disorder" don't fit for the "off-kilter" meaning. "The world has become askew" sorta feeling doesn't fit.

Reply:
Ugh yeah I was on the fence about this. I wanted to avoid using `world` here and on the next line and just keep it verbish, but not sure what a perfect analog for ズレる is and figured using a couple of words instead to hone in might help. 
> Contorting. Twisting. Shifting. Fracturing.

?

Reply:
yeah that works for me